manage.py

```diff
@@ -166,6 +166,9 @@ def pull_prod_data(default_user_password=None):
     print("Upgrading database from local migrations...")
     upgrade()
 
+    print("Refreshing materialized views...")
+    refresh_materialized_views()
```

Comment:
I wonder if we maybe want to drop and recreate the materialized views, rather than just refresh them? It might be that we're pulling data with a different structure. @frankieroberto thoughts?